_search-plugins/star-tree-index.md

```diff
@@ -39,26 +39,26 @@ The following image illustrates a standard star-tree index structure.
+Each node in a star-tree index points to a range of star-tree documents. Nodes can be further split into child nodes based on the [max_leaf_docs configuration]({{site.url}}{{site.baseurl}}/field-types/supported-field-types/star-tree/#stix-configuration-options). The number of documents that a leaf node points to is less than or equal to the number set by `max_leaf_docs`. This ensures that the maximum number of documents that need to traverse nodes to derive an aggregated value is at most the number of `max_leaf_docs`, which provides predictable latency.
 
 ### Star nodes
 
-Star nodes are children of non-leaf nodes that contain preaggregated records for data split after dimension removal, aggregating metrics for rows containing dimensions with identical values. These aggregated documents are then appended to the end of star-tree documents. If a document does contain a dimension with identical values, it traverses through the star node.
+Star nodes are nodes which contain the aggregated data of all the other nodes in the same dimension. This helps when querying the aggregated vales of a particular field without traversing through all the nodes in a cluster. 
 
-The star-tree index structure diagram contains the following three examples demonstrating how a document does or does not traverse star-tree nodes (indicated by the `*` symbol in the diagram) during a `Term` query, based on the average request size of the query and whether the document contains matching dimensions:
+The star-tree index structure diagram contains the following three examples demonstrating how a query behaves when retrieving aggregations from nodes in the star-tree, from both the `status` and `port dimensions` and star nodes (indicated by the `*` symbol in the diagram):
 
-- When the port equals `8443` and the status equals `200`. Because the status equals `200`, the query does not traverse through a star node, and the aggregated metric is stored at the end of a star-tree document.
-- When the status equals `200`. The query traverses through a star node in the `port` dimension because `port` is not present as part of the query.
-- When the port equals `5600`. The query traverses through a star node in the `status` dimension because `status` is not present as part of the query.
+- **Blue**: In a `terms` query searching for the average request size aggregation, the `port` equals `8443` and the status equals `200`. Because the query contains values in both the `status` and `port` dimensions, the query returns the aggregation from a non-star node.
```

Comment:
"""
Example of a `terms` query searching for the average request size aggregation, where the `port` equals `8443` and the status equals `200`. Because the query contains values in both the `status` and `port` dimensions, the query traverses via `200` status node and returns the aggregation from the `8443` child node.
"""